tv_apps/smart-home/_test/unit/card_manager_test.js

```diff
@@ -120,9 +125,10 @@ suite('smart-home/CardManager', function() {
       cardManager._reloadCardList().then(function() {
         assert.isTrue(cardManager._cardList.length > 0);
         assert.equal(cardManager._cardList[0].name, 'Television');
-      }, function(reason) {
+      }).then(done, done).catch(function(reason) {
```

Comment:
Will the following clause be executed? I thought if You rejected by done(), the catch() clause wouldn't be executed